GitUI/CommandsDialogs/WorktreeDialog/FormCreateWorktree.cs

```diff
@@ -170,15 +170,12 @@ private void UpdateWorktreePathAndValidateWorktreeOptions(object sender, EventAr
             void UpdateWorktreePath()
             {
                 var branchNameNormalized = NormalizeBranchName(radioButtonCheckoutExistingBranch.Checked
-                    ? ((IGitRef)comboBoxBranches.SelectedItem).Name
+                    ? ((IGitRef)comboBoxBranches.SelectedItem)?.Name ?? string.Empty
                     : textBoxNewBranchName.Text);
                 newWorktreeDirectory.Text = $"{_initialDirectoryPath}_{branchNameNormalized}";
             }
 
-            string NormalizeBranchName(string branchName)
-            {
-                return string.Join("_", branchName.Split(_invalidCharsInPath, StringSplitOptions.RemoveEmptyEntries)).TrimEnd('.');
-            }
+            string NormalizeBranchName(string branchName) => string.Join("_", branchName.Split(_invalidCharsInPath, StringSplitOptions.RemoveEmptyEntries)).TrimEnd('.');
```

Comment:
We have branch normalisation strategies we use in the branch dialog. Should we use the same?
Not asking to do it here, just making an observation.